packages/trader/scripts/release-prod.sh

```diff
@@ -32,6 +32,10 @@ if [[ ! $(git rev-parse --abbrev-ref HEAD) =~ master ]]; then
     exit 1
 fi
 
+if [[ -z $(command -v lerna) ]]; then
+    echo ${RED}"  > ERROR: "${RESET}"Please install lerna globally."
```

Comment:
```suggestion
    echo "$RED  > ERROR: $RESET Please install lerna globally."
```
Bash can handle this dynamic string. If it doesn't affect the functionality, I suggest the more readable approach.